.gitignore

```diff
@@ -5,3 +5,4 @@ vendor
 classes/log/error_log.txt
 composer.phar
 .idea/
+lib/*/docs/examples/*_test.json
```

Comment:
These files get generated by some of the PHPUnits. Now that the `.php` files with tests have been changed to the Moodle standard test filename (i.e. `*_test.php`) the generated test files have different filenames. 

The generation of these JSON files by the PHPUnits was already happening in `master`, and those automatically generated files have already been committed, e.g. see 1139e3f

I wasn't brave enough to take the decision to remove the existing ones! I thought it was safer/cleaner to `.gitignore` the new ones with the `*_test.json` extension.

```
+$ git diff master | grep -Pi '\$exampleFile'
-        $exampleFile = __DIR__.'/../docs/examples/'.$eventName.'.json';
-        file_put_contents($exampleFile, json_encode($output, JSON_PRETTY_PRINT));
-        $exampleFile = __DIR__.'/../docs/examples/'.$eventName.'.json';
-        file_put_contents($exampleFile, json_encode($output, JSON_PRETTY_PRINT));
-        $exampleFile = __DIR__.'/../docs/examples/'.$eventName.'.json';
-        file_put_contents($exampleFile, json_encode($output, JSON_PRETTY_PRINT));
+        $examplefile = __DIR__ . '/../../../lib/emitter/docs/examples/' . $eventname . '.json';
+        file_put_contents($examplefile, json_encode($output, JSON_PRETTY_PRINT));
+        $examplefile = __DIR__ . '/../../../lib/expander/docs/examples/' . $eventname . '.json';
+        file_put_contents($examplefile, json_encode($output, JSON_PRETTY_PRINT));
+        $examplefile = __DIR__ . '/../../../lib/translator/docs/examples/' . $eventname . '.json';
+        file_put_contents($examplefile, json_encode($output, JSON_PRETTY_PRINT));
```